meerevm/cmd/accountcmd.go

```diff
@@ -288,6 +293,33 @@ func accountImport(ctx *cli.Context) error {
 	if err != nil {
 		utils.Fatalf("Could not create the account: %v", err)
 	}
-	fmt.Printf("Address: {%x}\n", acct.Address)
+	fmt.Printf("PKAAddress: {%x}\n", acct.Address)
+	addrs, err := GetQngAddrsFromPrivateKey(hex.EncodeToString(key.D.Bytes()), params.ActiveNetParams.Params)
+	if err != nil {
+		utils.Fatalf("Could not create the account: %v", err)
+	}
+	for _, addr := range addrs {
```

Comment:
the `range` is strange anyway here, if you only need to show the pk-addr and pkh-addr